pkg/mappings.bzl

```diff
@@ -24,14 +24,16 @@ the following:
 - `pkg_filegroup` creates groupings of above to add to packages
 
 Rules that actually make use of the outputs of the above rules are not specified
-here.  TODO(nacl): implement one.
+here.
 """
 
 load("@bazel_skylib//lib:paths.bzl", "paths")
 load("//:providers.bzl", "PackageDirsInfo", "PackageFilegroupInfo", "PackageFilesInfo", "PackageSymlinkInfo")
 
 _PKGFILEGROUP_STRIP_ALL = "."
 
+REMOVE_BASE_DIRECTORY = ""
```

Comment:
Does this all still work if we made this an impossible value, like  chr(0)

Reply:
Not without modifications elsewhere.  Let me see how much it would take.

Reply:
It turned out to be pretty easy.  PTAL.